### ISSUE TITLE
Window actions using Touch

### ISSUE THREAD

**Issue by FriederHannenheim** [labwc/labwc#1183]
I have a Laptop with a touchscreen. I wish I could raise/move/resize windows by touching. The same way it's working with a mouse.

**Comment by FriederHannenheim** [comment]
Also closing/minimizing/maximizing by touching the corresponding buttons

**Comment by Consolatis** [comment]
We don't currently support mouse emulation for touchscreen devices. If the application supports the wayland touch protocol it should work for them though, e.g. gestures and friends.

**Comment by FriederHannenheim** [comment]
Yes. Touch works in applications. I want it to work for window management too
